dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/sequence/SequenceReader.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.sequence;
+
+import java.util.NoSuchElementException;
+
+/**
+ * Interface for readers than returns nucleotide subsequences of a given sequence source (like a fasta file or a WS)
+ */
+public interface SequenceReader {
```

Comment:
Do we really need this? The dbSNP importer is going to pull sequences from indexed FASTA files only.

Reply:
When I started implementing this I thought that maybe there were some ENA or Ensembl WS that could be used for that. Having to check every single variant individually, a JSON WS is not going to be good solution, so I can remove this interface. I don't mind to keep it either... anybody else has a opinion?